test/e2e/simple_bridge_and_bond.go

```diff
@@ -90,49 +99,17 @@ var _ = Describe("NodeNetworkState", func() {
 				updateDesiredState(br1Up)
 			})
 			AfterEach(func() {
-
-				// First we clean desired state if we
-				// don't do that nmstate recreates the bridge
-				resetDesiredStateForNodes()
-
-				// TODO: Add status conditions to ensure that
-				//       it has being really reset so we can
-				//       remove this ugly sleep
-				time.Sleep(1 * time.Second)
-
-				// Let's clean the bridge directly in the node
-				// bypassing nmstate
-				deleteConnectionAtNodes("eth1")
-				deleteConnectionAtNodes("eth2")
-				deleteConnectionAtNodes("br1")
-			})
-			It("should have the linux bridge at currentState", func() {
+				updateDesiredState(br1Absent)
 				for _, node := range nodes {
-					interfacesNameForNode(node).Should(ContainElement("br1"))
+					interfacesNameForNode(node).ShouldNot(ContainElement("br1"))
```

Comment:
Don't we need eventually here?

Reply:
It's inside "interfacesNameForNodes" it returns and AsyncAssertion